custom_dc/customize_ui.md

```diff
@@ -36,28 +36,30 @@ Note to use `-e custom` flag when starting local Flask server:
 Page header and footer can be customized in
 [base.html](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/base.html)
 by updating the html element within `<header></header>` and `<footer></footer>`.
+
 Homepage can be customized in
 [homepage.html](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/homepage.html).
 
 ### Update CSS and Javascript
 
 The custom Data Commons provides an
 [overrides.css](https://github.com/datacommonsorg/website/tree/master/static/custom_dc/custom/overrides.css)
-to override CSS styles. More style changes can be added to that file.
+to override CSS styles. It has a default color override. More style changes can
+be added in that file.
 
-To add more site customization, can put CSS and Javascript files under folder
+If there are already existing CSS and Javascript, put them under folder
```

Comment:
nit: under the [/static/custom_dc/custom](...) folder

Reply:
Done